spec/internal/config/storage.yml

```diff
@@ -0,0 +1,3 @@
+test:
+  service: Disk
+  root: /home/helpotters/dev/bounties/readyset-rails/tmp/storage
```

Comment:
Hmm this probably shouldn't be a hard-coded path -- is there a way to make this more general?

Reply:
Deleting it seems to be fine! Might autogenerate, but I've placed it in `.gitignore` in the meantime.